p2p/src/net/mock/transport/impls/stream_adapter/wrapped_transport/wrapped_listener.rs

```diff
@@ -29,6 +29,8 @@ use crate::{
     Result,
 };
 
+const MAX_CONCURRENT_HANDSHAKES: usize = 100;
```

Comment:
This setting should be moved somewhere else. Please throw in a TODO above it as I don't care right now.

Reply:
Sure, added.